pages/database-management/server-stats.md

```diff
@@ -36,6 +36,7 @@ The result will contain the following fields:
 | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
 | memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
 | peak_memory_res              | Peak RAM memory usage in the system during the whole run.                                        |
+| unreleased_delta_objects     | The current amount of unreleased objects with the information about the changes that write transactions have made, called Delta objects. Delta objects will eventually be deallocated from the system.                        |
```

Comment:
Avoid using a term to define itself. In this case "unreleased".
What are the units for this? KB, GB, MB?

Reply:
just a number of non-deallocated objects